products/health-care/digital-health-modernization/product/roadmap.md

```diff
@@ -1,9 +1,15 @@
 # DRAFT Digital Health Modernization Product Roadmap
```

Comment:
**General feedback**

- Confirming that these are meant to be epics, not PBIs? 
- Let’s add visual affordance to the teams that will be impacted under each epic when we present
- I think there's work we can do with MyVA in the "now." I don't feel strongly as to whether this needs its own piece of work or if we need to call this out more explicitly in the "optimize a single VA.gov health page" work. 
- Would like to discuss order of operations as it relates to the big 4 health tools -- you give examples of Rx and SM and I wonder if we should come into tomorrow's meeting with a strong POV. 
- Do we want to include work for technical discovery on replatforming health tools as step 1 (and/or other user research next steps)?